Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN apt-get -qq update && apt-get -qq install -y \
 
 # Really weird bug, setting LD_LIBRARY_PATH breaks pip, so run all the pip stuff before setting that.
 COPY requirements.txt ./requirements.txt
+
+
 RUN pip3 install -r requirements.txt
 
 RUN rm -rf /root/.cache/pip && \
```

Comment:
Dockerfile Protip:
remove desired files in the layer that created them

RUN pip3 install -r requirements.txt &&  rm -rf /root/.cache/pip 

This will reduce the actual size of the layer.

I just learnt this..... :(